src/internal/styles/global-vars.scss

```diff
@@ -0,0 +1,7 @@
+/*
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+$stickyVerticalTopOffset: --awsui-sticky-vertical-top-offset;
+$stickyVerticalBottomOffset: --awsui-sticky-vertical-bottom-offset;
```

Comment:
No code generation for this file, because introducing any global var is a big deal and it should not be easy.

Manual steps here is a feature, not an issue